policy/modules/system/logging.te

```diff
@@ -564,6 +566,8 @@ ifdef(`init_systemd',`
 
 	systemd_manage_journal_files(syslogd_t)
 	systemd_watch_journal_dirs(syslogd_t)
+	systemd_dontaudit_relabelfrom_journal_files(syslogd_t)
+	systemd_dontaudit_relabelto_journal_files(syslogd_t)
```

Comment:
I'm not dontaudit is the right choice. I've very recently seen some relabeling on a system that has the journal on tmpfs until the writable storage is mounted. However, I haven't had a chance to fully investigate.

Reply:
I dontaudited these because I didn't see any misbehavior on my systems if the access was denied, but I agree I think `allow` is better here.